### DISCUSSION TITLE
Request list of pending orders from mostro

### DISCUSSION THREAD

**Discussion by arkanoider** [MostroP2P/mostro#2] [Ideas]
Looking to twitter [video](https://twitter.com/negrunch/status/1604841310682316800) I was thinking if it could be nice to have an Action command which request directly all ( or a user defined number) the pending order nicely formatted.

So:
```mermaid
sequenceDiagram
    Client->>Mostro: request offers list
    Mostro->>Client: Order 1(Sell 1000 sats for 100 Eur )  
    Mostro->>Client: Order 2(Sell 1500 sats for 120 Usd ) 
```

Let me know what you think. In case I'd like to implement it.

**Comment by grunch** [comment]
Hi, yes, that could be done already and you can see it on the video, clients just get that data and show it to users
